app/styles/cells.less

```diff
@@ -56,12 +48,12 @@
       }
     }
     &.empty-value {
-      color: @color-text-lighter;
+      color: var(--color-gray-300);
```

Comment:
`500` instead ? as in the templates, we would have used `font-color-gray-500` for an empty state